### PR TITLE
Edit devops

### PR DESCRIPTION
**Pull request by onewildgamer**

### CODE REVIEW

**Comment by onewildgamer** [comment]
Please review and accept

File: devops.txt [hunk 1]
```diff
@@ -5,3 +5,5 @@ edit4
 edit3
 edit2
```
Comment (baari) [inline devops.txt:6]: change this

File: devops.txt [hunk 2]
```diff
@@ -5,3 +5,5 @@ edit4
...
 test file
+
```
Comment (baari) [inline devops.txt]: remove this line

File: devops.txt (hunk 1 above)
Comment (onewildgamer) [reply devops.txt:6]: not necessary for this change

**Review by baari — CHANGES_REQUESTED** [review]

File: devops.txt (hunk 2 above)
Comment (onewildgamer) [reply devops.txt]: Have removed in the latest commit

**Review by baari — APPROVED** [review]
looks good.

**Pull request merged**